keyserver/README.md

```diff
@@ -0,0 +1,56 @@
+# CONIKS Server implementation in Golang
+
+## Usage
+```
+⇒  go install github.com/coniks-sys/coniks-go/keyserver/coniksserver
+⇒  coniksserver -h
+/----------------------------------------------\
+|                  ,,:::::,:,,                 |
+|             :irrsrrrrrrrrrrrsrr:,            |
+|          irsrr::::,::::::::::iirrsi,         |
+|       ,rsr;:::irss5SXX9SXH2ssi:,:irrs:       |
+|      rrri::rsX9GGG99XXS9X99GGG9Hr:,:;rr:     |
+|    ,sri::r9GG9X252SSXXXXXSS25H99M9s::irri    |
+|    sr;::5M99H259MBB@@@B@B@BMGX2HS9GGr:i;ri   |
+|   rr;::XMXXH5G@B@BBMBMBMBM@B@BBX5SX9Mr:irs,  |
+|   sri:sMXXS5M@BBMBMMMMMMMBMBM@B@SSS9GG:iis:  |
+|   sii:2G9SS5@BBMMMMMMMBMMMMMBMBB92XS9Gi:rri  |
+|   rri:iM995HM@MBMMMBMMMBMBMBB@BM5SS9M5,iis,  |
+|   :s;i:rMGX5@BMMMM@B@B@B@@@B@MX2SX9MX::irr   |
+|    :ri::iSG9B@BBM@GGGMMMGG9X222X9M9s,:irs    |
+|     riii:,r2@B@B@B9252525252SS99Xs:,irsr     |
+|    ;riiiii:,:s5MB@M9GGGGXXSXS9H:,::rrs:      |
+|    sri;irrrii,,,::iirrriX9XS9GS,;rsr:        |
+|   :srrrsrr:irrrrii:i:::,9MGMM@Mii:           |
+|   2sr::,      ,:iirrrrriMMGSsr:              |
+|  ,,                                          |
+\----------------------------------------------/
```

Comment:
That's Tor Messenger specific.  We should keep this to CONIKS only.  Since there's no logo, what if you just did "CONIKS" in ascii art?

Pick your favourite http://patorjk.com/software/taag/#p=display&f=Rectangles&t=CONIKS

```
                                     
 _____ _____ _____ _____ _____ _____ 
|     |     |   | |     |  |  |   __|
|   --|  |  | | | |-   -|    -|__   |
|_____|_____|_|___|_____|__|__|_____|
                                     
```